rpc/server_test.go

```diff
@@ -95,9 +95,9 @@ func TestUnregisteredMethod(t *testing.T) {
 
 	// Sending an invalid method fails cleanly, but leaves the connection
 	// in a valid state.
-	_, err := sendRPC(opts, []net.Addr{s.Addr()}, nodeContext, "Foo.Bar",
-		&proto.PingRequest{}, &proto.PingResponse{})
-	if !testutils.IsError(err, ".*rpc: couldn't find method: Foo.Bar") {
+	if _, err := sendRPC(
+		opts, []net.Addr{s.Addr()}, nodeContext, "Foo.Bar", &proto.PingRequest{}, &proto.PingResponse{},
+	); !testutils.IsError(err, ".*rpc: couldn't find method: Foo.Bar") {
```

Comment:
This one, on the other hand, bothers me. A non-trivial (multi-line) assignment statement with a non-trivial (not `err != nil` or a handful of other common patterns) condition is hard to read, and I'd rather have a wider scope for the `err` variable than jam everything into one statement. This would be a case where additional braces for scoping might make sense.